plugins/kubernetes/test/controllers/kubernetes_dashboard_controller_test.rb

```diff
@@ -36,30 +44,112 @@
 
   private
 
-  def api_response
-    response = JSON.parse(parse_json_response_file('kubernetes_pod_api_response'))
-    response['items'][0]['metadata']['labels']['role_id'] = kubernetes_roles(:app_server).id.to_s
-    response['items'][1]['metadata']['labels']['role_id'] = kubernetes_roles(:resque_worker).id.to_s
-    response['items'][0]['metadata']['labels']['release_id'] = kubernetes_releases(:test_release).id.to_s
-    response['items'][1]['metadata']['labels']['release_id'] = kubernetes_releases(:second_test_release).id.to_s
-    JSON.generate(response)
-  end
-
   def compare_json(expected, actual)
     expected_json = JSON.parse(expected)
     actual_json = JSON.parse(actual)
-    [expected_json, actual_json].each do |json|
-      delete_ids_from json # drop IDs, they're generated
-    end
-    actual_json.must_equal expected_json
+    expected_json.must_equal actual_json
```

Comment:
If I remember correctly, the previous version of this assertion was intentional, so that the test output in case of a failure correctly identifies the expected and actual values.